mockgcp/mocknetapp/backupvault.go

```diff
@@ -102,11 +99,10 @@ func (s *backupVaultsService) UpdateBackupVault(ctx context.Context, req *pb.Upd
 	if err := s.storage.Update(ctx, fqn, obj); err != nil {
 		return nil, err
 	}
-	metadata := &pb.OperationMetadata{}
-	return s.operations.StartLRO(ctx, name.String(), metadata, func() (proto.Message, error) {
-		return obj, nil
-	})
+
+	return s.operations.DoneLRO(ctx, name.String(), &pb.OperationMetadata{}, obj)
```

Comment:
Same here. It looks like `StartLRO` should be a better mock for the real behavior?